include/koa/authentication/passport-strategies/salesforce_authentication_strategy.js

```diff
@@ -0,0 +1,74 @@
+const CustomStrategy = require('passport-custom').Strategy;
+const strategyServices = require('./strategy_services');
+const request = require('request-promise');
+const Promise = require('bluebird');
+const SALESFORCE_OAUTH_TOKEN_URL = process.env.SALESFORCE_OAUTH_TOKEN_URL || '';
+const SALESFORCE_OAUTH_AUTHORIZE_URL = process.env.SALESFORCE_OAUTH_AUTHORIZE_URL || '';
+
+module.exports = (pb) => {
+    async function getSalesforceSettings(req) {
+        let pluginService = new pb.PluginService({
+            site: req.ctx.__site
+        });
+        pluginService = Promise.promisifyAll(pluginService);
```

Comment:
We should double check to make sure plugin service (ie pb object) is not promisified.  There is no need to double promisify it if it is already promisified.

Reply:
It doesn't affect at all the site? I can save the promisified service by site, is that okay?